### PR TITLE
bh-1001 text scaling fix

### PR DESCRIPTION
**Pull request by sjv01180**

### CODE REVIEW

File: sass/main.scss [hunk 1]
```diff
@@ -423,11 +423,25 @@ footer {
     }
 }
 
+@media screen and (max-width: 1440px) {
+    #hero {    
+        #bh7 {
+            margin-top: -217px;
```
Comment (peterkos) [inline sass/main.scss:429]: Arbitrary number here -- could you use something more like `-200px`? 
However this still clips on my 16" mac, making the window sightly narrower.

File: sass/main.scss [hunk 2]
```diff
@@ -423,11 +423,25 @@ footer {
...
+        }
+        #hero-text {
+            width: 57%;
```
Comment (peterkos) [inline sass/main.scss:432]: Width of the other div in the flex container wasn't adjusted -- also `57%` seems arbitrary

File: sass/main.scss [hunk 3]
```diff
@@ -423,11 +423,25 @@ footer {
...
+}
+
 @media screen and (max-width: 1200px) {
 
     // Scaling idea, key is clip
     #shelf {
         width: 800px;
         overflow: clip;
     }
+
```
Comment (peterkos) [inline sass/main.scss:445]: nit: no added newlines here

**Review by peterkos — CHANGES_REQUESTED** [review]
Doesn't quite work :c 

Also please remove the `sass/.main.scss.swp` and `.index.html.swp` files! 

Screen Shot 2020-12-10 at 6 41 48 PM

**Comment by sjv01180** [comment]
Made some changes to the left padding on the hero id for supporting smaller laptop screens. 

If this works, we should consider adding additional variables for each screen size. This would give us a head start towards responsive web design.

**Review by peterkos — APPROVED** [review]
Great work Sam!

I've gone in and made a couple tweaks to better fit our design. You're absolutely right and we should definitely have vars for breakpoints -- I'm still trying to figure out the best way to plan adding mobile support (i.e., section-by-section, or just having it be done in blocks of work)

**Pull request merged**